cffdump/cffdump.c

```diff
@@ -2839,7 +2832,6 @@ static int handle_file(const char *filename, int start, int end, int draw)
 			printl(1, "test: %s\n", (char *)buf);
 			break;
 		case RD_CMD:
-			is_blob = true;
```

Comment:
maybe move it under `RD_TEST` case instead?  That packet will only show up traces from the freedreno/tests-3d stuff (which has some extra logging), but it shouldn't show up in $debugfs/rd traces?

Reply:
Sounds like this MR was dropped in favor of f669cae33cdc ("crashdec: add option to decode string markers")